sunpy/net/jsoc/jsoc.py

```diff
@@ -522,13 +525,10 @@ def get_request(self, requests, path=None, overwrite=False, progress=True,
         dl_set = True
         if not downloader:
             dl_set = False
-            downloader = Downloader(progress=progress, overwrite=overwrite, max_conn=max_conn)
-
-        if downloader.max_conn * kwargs['max_splits'] > 10:
-            warn_user("JSOC does not support more than 10 parallel connections. " +
-                      f"Changing the number of parallel connections to {2 * self.default_max_conn}.")
-            kwargs['max_splits'] = 2
-            downloader.max_conn = self.default_max_conn
+            # Private communication from JSOC say we should not use more than one connection.
+            if max_conn != self.default_max_conn:
+                log.info(f"max_conn keyword was passed and set to 1.")
```

Comment:
This message isn't very clear. Something like "Setting max parallel downloads to 1 for JSOC client." would be better

Reply:
Fixed.